components/script/dom/window.rs

```diff
@@ -996,7 +996,7 @@ impl Window {
                 page_clip_rect: self.page_clip_rect.get(),
             },
             document: self.Document().upcast::<Node>().to_trusted_node_address(),
-            document_stylesheets: document.stylesheets().clone(),
```

Comment:
It does clone the vector because of the Ref deref. To clone the ref you have to use Ref::clone